lib/statsLogger.js

```diff
@@ -122,6 +123,17 @@ exports.createInstance = function(flushInterval, backend, backendConfig, stats)
   };
 
   /**
+   * Record a stat once per flush, just before the flush is executed.  
+   *
+   * @param name The name of the stat to record
+   * @param statFunction A function that returns the value to record to the stat
+   */
+  StatsLogger.prototype.captureStatOnFlush = function(name, statFunction){
```

Comment:
Is there a need to add a corresponding method to remove an existing capture function?